src/main/java/gregtech/api/unification/material/materials/ElementMaterials.java

```diff
@@ -346,7 +351,7 @@ public static void register() {
 
         Iron = new Material.Builder(51, gregtechId("iron"))
                 .ingot().fluid().plasma().ore()
-                .color(0xC8C8C8).iconSet(METALLIC)
+                .color(0xC8C8C8, false).iconSet(METALLIC)
```

Comment:
Is this supposed to be adding a `false` here? I don't see a custom texture for Iron fluid being registered in MetaFluids